Sources/WhisperKit/Core/WhisperKit.swift

```diff
@@ -177,7 +177,7 @@ public class WhisperKit {
 
         // If a local model folder is provided, use it; otherwise, download the model
         if let folder = modelFolder {
-            self.modelFolder = URL(fileURLWithPath: folder)
+            self.modelFolder = URL(fileURLWithPath: folder) 
```

Comment:
```suggestion
            self.modelFolder = URL(fileURLWithPath: folder)
```

Should be able to commit this directly